src/core/ColorPicker/ColorPalette.tsx

```diff
@@ -160,7 +160,9 @@ export const ColorPalette = React.forwardRef(
                     onChangeComplete?.(color);
                     setActiveColor(color);
                   }}
-                  isActive={color.equals(activeColor)}
+                  isActive={
+                    color.toHslString(true) === activeColor.toHslString(true)
```

Comment:
why changed?

Reply:
I'm not sure but equals wasn't working correctly with alpha values, with toHslString(true) it does work with the alpha value so it sets the ColorSwatch active when the alpha values match
![opacity-Active](https://user-images.githubusercontent.com/88331924/139695539-686aaddd-cbcd-41ae-8e0a-c594b7b29d55.gif)

Reply:
Have you tried debugging to find out why `equals` doesn't work? Might need to fix that

Reply:
Looks like its just because of all the color conversions. Sometimes the tbgr value is slightly off so equals doesn't work, but 2 slightly different tbgr values will give the same string value.